test/manual/kerberos.test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+const { MongoClient } = require('../../src');
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('Kerberos', function () {
+  if (process.env.MONGODB_URI == null) {
+    console.log('skipping Kerberos tests, MONGODB_URI environment variable is not defined');
+    return;
+  }
+  let krb5Uri = process.env.MONGODB_URI;
+  if (process.platform === 'win32') {
+    console.log('Win32 run detected');
+    if (process.env.LDAPTEST_PASSWORD == null) {
+      throw new Error('The env parameter LDAPTEST_PASSWORD must be set');
+    }
+    const parts = krb5Uri.split('@', 2);
+    krb5Uri = `${parts[0]}:${process.env.LDAPTEST_PASSWORD}@${parts[1]}`;
+  }
```

Comment:
In the original disabled tests, there were separate copies of the tests for Win32, but adding the password into the URL from the environment variable was the only difference between them. I added this logic to preserve the original logic, but haven't actually tried running the tests on Windows, since we don't currently have a win32 variant.